src/_pytest/python.py

```diff
@@ -1085,7 +1085,7 @@ def get_direct_param_fixture_func(request: FixtureRequest) -> Any:
 
 
 # Used for storing pseudo fixturedefs for direct parametrization.
-name2pseudofixturedef_key = StashKey[dict[str, FixtureDef[Any]]]()
+name2pseudofixturedef_key = StashKey[dict[str, FixtureDef[Any, Any]]]()
```

Comment:
I'm starting to get the impression we need a type alias for those dicts as they land in too many places

Reply:
Done.